website/scripts/build-static-content.js

```diff
@@ -512,6 +512,160 @@ module.exports = {
+            let tableMdString = '\n## '+table.name;
+            if(_.endsWith(table.name, '_events') || table.evented){
+              // If the table name ends in `_events` or has `"evented": true`, we'll add an evented table label (in html)
+              tableMdString += '   <div purpose="evented-table-label"><span>EVENTED TABLE</span></div>\n';
+            }
+            // Add the tables description to the markdown string and start building the table in the markdown string
+            tableMdString += '\n\n'+table.description+'\n\n|Column | Type | Description |\n|-|-|-|\n';
+
+            // Iterate through the columns of the table, we'll add a row to the markdown table element for each column in this schema table
+            for(let column of table.columns) {
+              if(!column.hidden) { // If te column is hidden, we won't add it to the final table.
+                let columnDescriptionForTable = column.description;// Set the initial value of the description that will be added to the table for this column.
+                if(typeof columnDescriptionForTable === 'String'){ // Replacing pipe characters with an html entity in column descriptions to keep it from breaking markdown tables.
+                  columnDescriptionForTable = columnDescriptionForTable.replaceAll('|', '&#124;');
+                }
+                keywordsForSyntaxHighlighting.push(column.name);
+                if(column.required) { // If a column has `"required": true`, we'll add a note to the description that will be added to the table
+                  columnDescriptionForTable += '<br> **Required in `WHERE` clause** ';
+                }
+                if(column.requires_user_context) { // If a column has `"requires_user_context": true`, we'll add a note to the description that will be added to the table
+                  columnDescriptionForTable += '<br> **Defaults to root** ';
+                }
+                if(column.platforms) { // If a column has an array of platforms, we'll add a note to the final column description
+
+                  let platformString = '<br> **Only available on '; // start building a string to add to the column's description
+
+                  if(column.platforms.length === 2) { // Because there are only three options for platform, we can safely assume that there will be at most 2 platforms, so we'll just handle this one of two ways
+                    // If there are two values in the platforms array, we'll add the capitalized version of each to the columns description
+                    platformString += _.capitalize(column.platforms[0]).replace('Darwin', 'macOS') +' and '+ _.capitalize(column.platforms[1]).replace('Darwin', 'macOS');
+                  } else {
+                    // Otherwise, there is only one value in the platform array and we'll add that value to the column's description
+                    platformString += _.capitalize(column.platforms[0]).replace('Darwin', 'macOS');
```

Comment:
normalize platform to "macOS" above when finalizing the Fleet schema
(I know this probably wasn't in the spec, just a realization)